lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -113,6 +114,55 @@ describe('FilePickerApp', () => {
     });
   }
 
+  context('when create assignment configuration is enabled', () => {
+    const authURL = 'https://testlms.hypothes.is/authorize';
+    const path = '/api/canvas/assignments';
+    let fakeAPICall;
+    let fakeNewAssignment;
+
+    beforeEach(() => {
+      fakeConfig.filePicker.createAssignmentAPI = {
```

Comment:
I'm trying to set the config value for this:

https://github.com/hypothesis/lms/blob/0fb537ecdf423d414a5be470cf76a1d55879bd9e/lms/static/scripts/frontend_apps/components/FilePickerApp.js#L66

to latter assert an api call is made based on this conditional:

https://github.com/hypothesis/lms/blob/0fb537ecdf423d414a5be470cf76a1d55879bd9e/lms/static/scripts/frontend_apps/components/FilePickerApp.js#L117

but I don't think I'm setting it right, I might be typo I can't spot or something more fundamentally. 

The rest of the test might be wrong as well but I haven't got as far as I'm doing something wrong on the setup part.